package.json

```diff
@@ -44,27 +44,35 @@
     "git-hooks": "^1.0.2",
     "grunt": "^0.4.5",
     "grunt-bump": "^0.7.0",
-    "grunt-casperjs": "^2.1.0",
     "grunt-contrib-concat": "^1.0.0",
     "grunt-contrib-connect": "^1.0.0",
     "grunt-contrib-uglify": "^1.0.0",
     "grunt-contrib-watch": "^1.0.0",
     "grunt-jscs": "^2.8.0",
     "grunt-mocha": "^0.4.15",
+    "grunt-webdriver": "just-boris/grunt-webdriver#infinite-loop",
```

Comment:
There’s a plan to move to Gulp anyway. Will it solve this _here’s-my-fork_ complexity?